source/Tipster.cpp

```diff
@@ -62,15 +62,21 @@ Tipster::Tipster()
 
 	fTipsterTextView = new TipsterText();
 	fIcon = new BButton("iconview", "", new BMessage(OPEN_URL));
+	fIcon -> SetToolTip( B_TRANSLATE("Open the Tipster homepage") );
 
 	BDragger* dragger = new BDragger(this);
 
 	BLayoutBuilder::Group<>(this, B_HORIZONTAL, 0)
-			.SetInsets(-2, -2, 0, -2)
-			.Add(fIcon)
+			.SetInsets(1, 1, 1, 1)
+			.AddGroup(B_VERTICAL,0)
+				.Add(fIcon)
+				.AddGlue()
+				.End()
+
```

Comment:
No need for an empty line, I think. The indentation is obvious enough to distinguish separate elements.